challenges/serializer-api.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import data

```

Comment:
shouldn't this be `from .models-api import data`

Reply:
HS Challenge is the name of my project and within my project are 3 apps
1. hschallenge
2. info
3. api

This serializer.py file is located within my api app. Therefore, to reference models that are created within that app or any file within that app a period as well as the file/folder name is all that is required.

Reply:
I included the "-api" so that I could provide some distinction within my files. As requested, I've only submitted files that I wrote myself. Without including the "-api" there wouldn't be a way to distinguish between a model that has been created within my info app or a model in my api app. They are both titled models.py.

I guess I could have fixed this by using folders. I'm currently getting familiar with best practices within Github but any recommendation that you or Charles could provide regarding this would be greatly appreciated!